pkg/apis/serving/v1alpha2/transformer.go

```diff
@@ -17,6 +17,7 @@ const (
 // Transformer interface is implemented by all Transformers
 type Transformer interface {
 	GetContainerSpec() *v1.Container
+	GetStorageUri() string
```

Comment:
for now, there is no use case for transformer to need `GetStorageUri`

Reply:
Thanks Hougang.
Seen from spec there's CustomSpec in TransformerSpec and the function GetStorageUri is already implemented for CustomSpec. 
Also from code there's should always exist  storageUri for transformer to load the python transformer image. I just add it to the interface as the predictor and explainer do the same.

Reply:
`GetStorageUri` is used to download model file to container, for transformer, it doesn't need model file in local container.

Reply:
Read the code again, yes it doesn't need model file in local container. The image for transformer container is just specified as image and unless the user implemented transformer need storage to do something it's not needed. 
Since there's no this kind of use case now, agree we can remove it, thanks.